package.json

```diff
@@ -29,11 +29,14 @@
     "bitcore-mnemonic": "=1.1.1",
     "lisk-js": "=0.4.2",
     "moment": "=2.15.1",
+    "postcss": "^6.0.2",
```

Comment:
please save all dependencies with `=` instead of `^`, e.g. "postcss": "=6.0.2",`